js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/listIndices/CollectionIndexActionButtons.tsx

```diff
@@ -10,7 +10,7 @@ import { DeleteIndexModal } from "../DeleteIndexModal";
 export function CollectionIndexActionButtons({
   collectionIndex
 }: {
-  collectionIndex: Index;
+  collectionIndex: Index & { progress?: number };
```

Comment:
Q: does arangojs `Index` not have progress? And if so, can it/should it be added there?  @pluma4345